Client/Ecosia/Settings/EcosiaDebugSettings.swift

```diff
@@ -0,0 +1,209 @@
+        navigationController?.topViewController?.present(alert, animated: true) {
+            DispatchQueue.main.asyncAfter(deadline: .now() + 1.5) {
+                alert.dismiss(animated: true)
+            }
+        }
+    }
+}
+
+final class ResetSearchCount: HiddenSetting {
+    override var title: NSAttributedString? {
+        return NSAttributedString(string: "Debug: Set search count to 0", attributes: [NSAttributedString.Key.foregroundColor: UIColor.theme.tableView.rowText])
+    }
+
+    override var status: NSAttributedString? {
+        return NSAttributedString(string: "\(User.shared.treeCount)", attributes: [NSAttributedString.Key.foregroundColor: UIColor.theme.tableView.rowText])
+    }
+
+    override func onClick(_ navigationController: UINavigationController?) {
+        User.shared.treeCount = 0
+        self.settings.tableView.reloadData()
+        NotificationCenter.default.post(name: .HomePanelPrefsChanged, object: nil)
+    }
+}
+
+final class ChangeSearchCount: HiddenSetting {
+    override var title: NSAttributedString? {
+        return NSAttributedString(string: "Debug: Increase search count by 10", attributes: [NSAttributedString.Key.foregroundColor: UIColor.theme.tableView.rowText])
+    }
+
+    override var status: NSAttributedString? {
+        return NSAttributedString(string: "\(User.shared.treeCount)", attributes: [NSAttributedString.Key.foregroundColor: UIColor.theme.tableView.rowText])
+    }
+
+    override func onClick(_ navigationController: UINavigationController?) {
+        User.shared.treeCount += 10
+        self.settings.tableView.reloadData()
+        NotificationCenter.default.post(name: .HomePanelPrefsChanged, object: nil)
+    }
+}
+
+final class UnleashDefaultBrowserSetting: HiddenSetting {
+    override var title: NSAttributedString? {
+
+        return NSAttributedString(string: "Debug: Unleash default browser variant", attributes: [NSAttributedString.Key.foregroundColor: UIColor.theme.tableView.rowText])
+    }
+
+    override var status: NSAttributedString? {
+        let variant = Unleash.getVariant(.defaultBrowser).name
+
+        return NSAttributedString(string: "\(variant) (Click to reset)", attributes: [NSAttributedString.Key.foregroundColor: UIColor.theme.tableView.rowText])
+    }
+
+    override func onClick(_ navigationController: UINavigationController?) {
+        Task {
+            do {
+                _ = try await Unleash.reset(env: .staging)
+            } catch {
+                debugPrint(error)
+            }
+            await MainActor.run {
+                self.settings.tableView.reloadData()
+
+                let alert = AlertController(title: "New variant assigned", message: "It might be the old one though", preferredStyle: .alert)
+                navigationController?.topViewController?.present(alert, animated: true) {
+                    DispatchQueue.main.asyncAfter(deadline: .now() + 1.5) {
```

Comment:
Is "1.5" the time in seconds? Just curious

Reply:
Yes, seconds 👍